higgstautau/lephad/branches.py

```diff
@@ -0,0 +1,190 @@
+# branches to ignore in the input tree and not include in the output tree
```

Comment:
Moved all higgstautau/*/branches.py into higgstautau and adjusted the lines in all respective *skim.py